Agent/ProcessControl/ProcessCommands.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+import multiprocessing
+from AaSystem.LogAndPrint.Log import PrintRedAndLog
+
+
+Processes = {}
+
+
+def RunProcess(request):
+    print("running!")
+    # if processName in Processes:
+    #     PrintRedAndLog(f"Process {processName} is an active process. Cannot run multiple processes by the same name.")
+    #     return
+    # newProcess = multiprocessing.Process(target=func)
```

Comment:
check ProcessPoolExecutor which is very easy to use and as Futures, workers and etc.

Reply:
Will check but might be moving back to a single threaded implementation with timeStamp based activation